Kernel/Applications/spinner.cpp

```diff
@@ -10,19 +10,37 @@
  */
 #include <Applications/spinner.hpp>
 #include <Devices/Graphics/console.hpp>
+#include <Memory/Heap.hpp>
+#include <Library/string.hpp>
+#include <Arch/Memory.hpp>
+#include <Panic.hpp>
 
 namespace Apps {
 
 void spinner(void) {
     Console::printf("\n");
     int i = 0;
     const char spinnay[] = { '|', '/', '-', '\\' };
+    for (size_t size = 1; size < 1024 * 1024 * 1024; size++) {
+        void* ptr = malloc(size);
+        if (ptr == nullptr) {
+            panic("null malloc in stress test!");
+        }
+        memset(ptr, 0xCAFEBABE, size / sizeof(0xCAFEBABE));
```

Comment:
This is only writing one quarter of the memory allocated since `memset` only writes bytes of data